assets/questions/sortItinerary.js

```diff
@@ -0,0 +1,137 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Sort Scrambled Itinerary.
+ */
+
+globalData.questions['sortItinerary'] = {  // eslint-disable-line dot-notation
+  title: 'Sort Scrambled Itinerary',
+  starterCode: {
+    python:
+`def sortItinerary(tickets):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def justConnectTickets(cls, s):
+        return "-".join(s.split(","))
+`
+  },
+  tasks: [{
+    instructions: [
+      [
+        'Whoops! Airline tickets for trips of your boss are messed up! ',
+        'But do not worry, the sortItinerary function can sort this out. ',
+        'It takes a string that represents some airline tickets as input. ',
+        'The string can be split by comma into substrings that looks like ',
+        '"XXX-YYY". This substring represents a ticket and means the plane ',
+        'departs from XXX and arrives at YYY, noticing that departure and ',
+        'destinations are not necessarily of length 3. ',
+        'As an example, "LAX-JFK,JFK-ATL" means the itinerary has two flights',
+        ': one from LAX to JFK and the other from JFK to ATL. '
+      ].join(''),
+      [
+        'The function needs to return a string of the sorted itinerary. ',
+        'Locations need to be connected by dash. For the above example, ',
+        'the return string should be "LAX-JFK-ATL". '
+      ].join(''),
+      [
+        'All tickets must be used to form an itinerary. ',
+        'For now, assume there is no loops in the itinerary and is guaranteed ',
+        'to have one and only one itinerary given the list of tickets. '
+      ].join('')
+    ],
+    prerequisiteSkills: ['Graph', 'String', 'String Manupulation'],
+    acquiredSkills: ['Graph Traverse', 'Topological Sort'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'sortItinerary',
+    correctnessTests: [{
+      input: 'LAX-JFK',
+      allowedOutputs: ['LAX-JFK']
+    }, {
+      input: 'JFK-ATL,LAX-JFK',
+      allowedOutputs: ['LAX-JFK-ATL']
+    }, {
+      input: 'MUC-LHR,JFK-MUC,SFO-SJC,LHR-SFO',
+      allowedOutputs: ['JFK-MUC-LHR-SFO-SJC']
+    }],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.justConnectTickets',
+      messages: [
+        "Tickets order are scrambled. Don't just connect them in given order. ",
```

Comment:
Try to make your hints progress from vaguer to more specific / targeted feedback. Look at other buggyOutputTests in the repo for examples.